nasa-gibs/worldview
# WV-2628: Metadata Misspelled

## Description
If you select the **Sea Surface Height Anomalies (L4) layer** in the layer description you will see `NoSuchKey The specified key does not exist`.

![image](https://user-images.githubusercontent.com/19156576/226427002-428dd9f7-e63b-45cc-a964-928f1c6bbf21.png)

Metadata for this layer is not linked correctly because the filename(s) are misspelled.

Fixes # WV-2628

## How To Reproduce
- Add layers & search for "Sea Surface Height "
- Highlight/Select "Sea Surface Height Anomalies (L4)" layer
- Confirm the error within the layer description below the image

## How To Test
- git checkout wv-2628-meta-mispelling
- npm install
- npm run watch
- Add layers & search for "Sea Surface Height "
- Highlight/Select "Sea Surface Height Anomalies (L4)" layer
- Confirm the metadata description is loading properly

@nasa-gibs/worldview
